src/lib/trace-ir/field-class.c

```diff
@@ -669,6 +671,23 @@ add_mapping_to_enumeration_field_class(struct bt_field_class *fc,
 	BT_LIB_LOGD("Added mapping to enumeration field class: "
 		"%![fc-]+F, label=\"%s\"", fc, label);
 
+	// See if this remains a bit field enum field
+	if (enum_fc->is_bit_field_enum) {
+		if (range_set->ranges->len != 1) {
```

Comment:
What is this doing ? Does it exclude enums which have duplicated labels for the same bit ? If it is the case, is it really what we want here ?